README.rst

```diff
@@ -1,18 +1,10 @@
 Unpadded Base64
 ===============
 
-.. image:: https://img.shields.io/pypi/v/unpaddedbase64.svg
-    :target: https://pypi.python.org/pypi/unpaddedbase64/
-    :alt: Latest Version
```

Comment:
I do like shields, but indeed the one for `unpaddedbase64` seems to be broken.

`matrix-synapse` is fine though (![](https://img.shields.io/pypi/v/matrix-synapse)), which makes me wonder if the shield will be fixed after we fix packaging :shrug:

Reply:
It very well might, but I'm also a bit wary of relying on external resources for a single-file, pure-python library that we're frankly unlikely to ever touch again ;)